proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -27,9 +27,13 @@ message ConsumerAdditionProposal {
     // the proposed initial height of new consumer chain.
     // For a completely new chain, this will be {0,1}. However, it may be different if this is a chain that is converting to a consumer chain.
     ibc.core.client.v1.Height initial_height = 4 [(gogoproto.nullable) = false];
-    // genesis hash with no staking information included.
+    // genesis hash contains initial chain genesis state without the cross-chain validating validator set
+    // it is used for off-chain confirmation of genesis.json validity by validators and other parties.
+    // i.e. hashing the genesis.json provided by the consumer chain project staff should produce this value.
```

Comment:
```suggestion
    // The hash of the genesis state needed for the consumer CCV module. 
    // It is used for off-chain confirmation of genesis.json validity by validators and other parties.
```
AFAIK it does contain also the initial valset, see https://github.com/cosmos/interchain-security/blob/87c21be74f121802d8be170d1275130466a9be59/x/ccv/provider/keeper/proposal.go#L277

Reply:
True. I'm drawin my conclusions from Game of Chains testnet.

In the testnet the proposals were handled as follows:
1. the genesis.json hashed on proposal submission did not include any ccv state (no `initial_val_set` in genesis.json)
2. after spawn time, the genesis.json `initial_val_set` was populated and that was the genesis file used to start the chain.


Since the initial_val_set on proposal submission can be different that the actual valset at spawn time, does it make sense to have the `initial_val_set` set to anything but an empty array?

On spawn time the valset may be different, i.e. the validators on the provider may change - does that bring up any issues?

Reply:
Ah yes, that's true. The `genesis_hash` cannot contain any info about the CCV module. It's the hash of the genesis state without the CCV module data, as that part is computed by `MakeConsumerGenesis` when the proposal is handled by the provider chain.

Reply:
I've updated my suggestion above

Reply:
Done